.github/workflows/node.js.yml

```diff
@@ -0,0 +1,70 @@
+name: fail_fast_CI
+on:
+  push:
+    branches:
+      - dev
+      - feature    
+      - ktchethan-patch-1
+  pull_request:
+  release:
+    types:
+      - staging
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  checkout_and_install:
+    strategy:
+      matrix:
+         node: ["15.14.0"]        
```

Comment:
Why only one node version in this file?

Reply:
Added only node version to test locally rather then running on multiple node versions and shorten the execution time as well.